.vscode/settings.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Remove this file please, I guess it was added by accident, thx

Reply:
Yes, this was added by accident.
to prevent project-settings from being accidently added `.vscode` was added in the git ignore (right below `.idea`)